src/NotificationsApp.vue

```diff
@@ -32,6 +33,15 @@
 					<transition-group class="notification-wrapper"
 						name="list"
 						tag="ul">
+						<Notification v-if="hasThrottledPushNotifications"
+							:key="-2016"
+							datetime="warning"
```

Comment:
datetime=warning ?

Reply:
Yeah, its a hack to identify this notification inside the component...